src/Codeception/Util/Shared/Namespaces.php

```diff
@@ -1,13 +1,25 @@
 <?php
+
+declare(strict_types=1);
+
 namespace Codeception\Util\Shared;
 
+use function array_filter;
+use function array_pop;
+use function array_shift;
+use function count;
+use function explode;
+use function implode;
+use function ltrim;
+use function str_replace;
+
 trait Namespaces
 {
     protected function breakParts($class)
     {
         $class      = str_replace('/', '\\', $class);
         $namespaces = explode('\\', $class);
-        if (count($namespaces)) {
+        if (count($namespaces) > 0) {
```

Comment:
PHPStorm always suggests to use `!empty()`

Reply:
Interesting, does that suggestion come from a plugin?